packages/data-explorer/src/main.ts

```diff
@@ -9,7 +9,9 @@ import i18n from '@molgenis/molgenis-i18n-js'
 Vue.config.productionTip = false
 
 Vue.use(i18n, {
-  namespace: ['filters'],
+  lng: 'en',
```

Comment:
oke for now be need to think about getting this from backend ( maybe via context ?)

Reply:
there is a sticky for that!